samples/type_ahead_search_adaptive_cards/csharp/README.md

```diff
@@ -0,0 +1,87 @@
+---
+page_type: sample
+description: Demonstrating feature of type ahead search (static and dynamic) control in Adaptive cards.
```

Comment:
Demonstrating the feature of type ahead search (static and dynamic) control in Adaptive Cards.